### DISCUSSION TITLE
Groundwater level setting inquiry

### DISCUSSION THREAD

**Discussion by wj3aj4vu4** [Anura3D/Anura3D_OpenSource#203] [Q&A] [answered]
How should I set the groundwater, considering that it is not uniformly horizontal as shown in the tutorial? 
The groundwater in the field varies in its horizontal distribution.
I intend to establish two groundwater tables to simulate the effects of a rising groundwater level. 
Is the setup similar to the one described in the tutorial 'Slope Failure Under Transient Hydraulic Loads'?"

The situ data is as shown in the figure below.
![GWL-in situ](https://github.com/Anura3D/Anura3D_OpenSource/assets/124948337/31304313-c6d9-4f5a-bce5-8d98e522b75d)

**Comment by FrancescaCeccato** [comment]
Hi, Anura3D implements fully coupled formulation, thus if you want to simulate the process of rising water table you should assign appropriate hydraulic boundary conditions to obtain the result. 
If th water table is only a starting condition of the simulation, then follow chapter 12 of the tutorial.
You cannot assign different water tables to different materials or chage water table during simulation.

**Reply by wj3aj4vu4** [reply]
How should I set up the initial groundwater level that I want to simulate?
In the tutorial, Chapter 12 depicts the groundwater level as a horizontal line, but mine fluctuates. Should I set the water table profile in the initial conditions or configure it from the file PSF_1?
Thanks for your answer.

**Reply by FrancescaCeccato** [reply]
A no-horizontal groundwater level can be set with a file or by drawing an additional line in the GiD model that you can assing as initial freatic surface. This will be used to calculate the initial water pressures (pressure=gamma_w*depth), while initial stresses are calculated with K0 procedure (sigma_y=gamma_soil*depth, sigma_x=K0*sigma_y). In this way stresses are not in equilibrium in a slope, therefore you need a gravity loading phase with appropriate boundary conditions that can mantain the initial water level that you prescribed.

**Reply by wj3aj4vu4** [reply]
Thank you for your previous response. I have established the initial groundwater level (created using .PSF_01). However, I want to simulate the groundwater level rising to a specific level. Should I use the removal of fixity and apply loading conditions in the initial conditions to apply water pressure to the initial groundwater level (similar to Chapter 11)?
Or another method that you can sugesst me.
Thank for your sincerely replying.

**Reply by FrancescaCeccato** [reply]
If you want so simulate water level rising than you have to apply appropriate boundary conditions that generate this. For example external water pressure at a boundary as in chap 11.
Note that this is similar to fully coupled FEM anlyses.

**Reply by wj3aj4vu4** [reply]
Does the Anura3D software support file types like CPS02 for representing groundwater levels as the second type? 
Also, when simulating the rise of groundwater levels by adjusting the "liquid pressure (linear)," should it be set on both sides, and do the fixities need to be removed? 
Despite trying multiple methods to simulate the rise of groundwater levels, the issue remains unresolved. 
I sincerely request assistance with this matter.

**Reply by wj3aj4vu4** [reply]
Also, when to use 'apply initial water pressure'?

**Reply by FrancescaCeccato** [reply, marked as answer]
At the moment it is not possible to change the groundwater level by assigning a new phreatic surface. The procedure only works at the first load step to initialize stresses.
Clearly in a boundary you can apply a fixity or a pressure, not both. This is identical to FEM.
"Apply initial water pressure" can be used to assign in the entire domani an initial value of pressure everywhere. Again this is an initialization procedure and is used only in the very first time step and cannot be applied in follow up load steps.
